### DISCUSSION TITLE
Licence 3rd Clause

### DISCUSSION THREAD

**Discussion by dizont** [pvlib/pvlib-python#1328] [General]
Hello,

Thank you for creating such a wonderful tool. I want to use your tool in order to build a broader solar energy tool, that includes solarthermic systems as well. I was reading the licence and while everything seems to be clear, the 3rd clause of the BSD Licence is a bit not clear. Could you explain what it means ?

In the end my tool will be integrated and be a part of a more general tool for economical calculations, which is going to be also open source.

**Comment by kandersolar** [comment]
Hi @dizont, thanks for your interest in pvlib!  As someone who is not a lawyer, I am reluctant to attempt to clarify the license language for fear of unintentionally changing the meaning in some way.  I speculate that the other maintainers feel similarly, though maybe not.  

In case it's not obvious: pvlib python is one of many projects using the 3-Clause BSD license and I'm sure a bit of searching the internet will turn up helpful material for how to interpret its meaning correctly.
